### PR TITLE
Add Polish (PL_pl) Localization

### PR DESCRIPTION
**Pull request by playsenge**
A simple Polish localization. Will definitely improve it in the future, since it's not perfect.

### CODE REVIEW

**Comment by ashmonty** [comment]
Thanks for your contribution!
If you don't mind, I'm gonna turn this into a draft so you can revert it into a pull when it's done

**Comment by playsenge** [comment]
Everything is translated. It is obviously very easy to understand, however some things may just have small errors, however I couldn't find such.

File: locales/PL_pl.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+				"github": "https://github.com/shutterbug2000"
+			},
+			{
+				"name": "Kinnay",
+				"special": "Specjalne podziękowania",
+				"caption": "Research w sprawie struktur danych Nintendo",
+				"picture": "https://cdn.discordapp.com/avatars/186572995848830987/b55c0d4e7bfd792edf0689f83a25d8ea.png?size=128",
+				"github": "https://github.com/Kinnay"
+			},
+			{
```
Comment (ashmonty) [inline locales/PL_pl.json:180]: NinStar's strings are missing in your locale. Could you add them?
```suggestion
			{
				"name": "NinStar",
				"caption": "Icons for the Mii Editor and Juxt reactions",
				"picture": "https://github.com/ninstar.png",
				"github": "https://github.com/ninstar"
			},
			{
```

File: locales/PL_pl.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+				"picture": "https://cdn.discordapp.com/avatars/191370953807233024/0311b61e2009c1576828dd2e9a59d72e.png?size=128",
+				"github": "https://github.com/shutterbug2000"
+			},
+			{
+				"name": "Kinnay",
+				"special": "Specjalne podziękowania",
+				"caption": "Research w sprawie struktur danych Nintendo",
+				"picture": "https://cdn.discordapp.com/avatars/186572995848830987/b55c0d4e7bfd792edf0689f83a25d8ea.png?size=128",
+				"github": "https://github.com/Kinnay"
+			},
```
Comment (ashmonty) [inline locales/PL_pl.json:179]: NinStar's strings are missing in your locale. Could you add them?
```suggestion
			{
				"name": "NinStar",
				"caption": "Icons for the Mii Editor and Juxt reactions",
				"picture": "https://github.com/ninstar.png",
				"github": "https://github.com/ninstar"
			},
			{
```

**Review by ashmonty — CHANGES_REQUESTED** [review]

File: locales/PL_pl.json (hunk 2 above)
Comment (playsenge) [reply locales/PL_pl.json:179]: I suck at GitHub PR management, but the translation should be:
"Ikony do edytora Mii oraz reakcji do Juxt"

**Pull request merged**